### DISCUSSION TITLE
IP Adapter XL + & IP Adapter Face

### DISCUSSION THREAD

**Discussion by anwoflow** [Mikubill/sd-webui-controlnet#2195] [General]
Want to ask about support for the updated IP Adapter XL model (ViT-h, plus version) and the IP Adapter face model 

links 
https://huggingface.co/h94/IP-Adapter/tree/main/sdxl_models
https://huggingface.co/h94/IP-Adapter/blob/5c2eae7d8a9c3365ba4745f16b94eb0293e319d3/models/ip-adapter-plus-face_sd15.bin

I'm guessing just putting the files in the folder won't be enough

**Comment by stonksCode** [comment]
I'd bump this too. I've been having issues with my notebook in sagemaker just crashing hard when attempting to use.

I've gotten the ViT-h plus model for SDXL. Controlnet seems to download a vit-h preprocessor when it loads in. When I use the two, it goes through like its going to start, the hook takes ~40 seconds and then it shows a progress bar... then fails out. No error message, no change in the WebUI. The notebook crashes in Sagemaker and locks up the actual JupyterNotebook to where even restarting the notebook breaks. It takes 15-20min for the whole thing to crash out so I can stop the Sagemaker instance and attempt to restart.
